enterprise/server/auth/auth.go

```diff
@@ -20,7 +20,7 @@ import (
 	"github.com/buildbuddy-io/buildbuddy/server/util/log"
 	"github.com/buildbuddy-io/buildbuddy/server/util/lru"
 	"github.com/buildbuddy-io/buildbuddy/server/util/random"
-	"github.com/buildbuddy-io/buildbuddy/server/util/request_context"
+	requestcontext "github.com/buildbuddy-io/buildbuddy/server/util/request_context"
```

Comment:
This would be better saved for a separate PR.
Instead of introducing an alias here, it might be better to change the package name to match the filename.

Reply:
Thank you for catching this. vim somehow auto adds it. Reverted